src/stc/completionitem.cpp

```diff
@@ -0,0 +1,11 @@
+#include "wx/stc/completionitem.h"
```

Comment:
It's probably not necessary to have this file at all, considering how trivial it is you can just make all the methods inline.

But if you do add a new source file, please use the usual format for it, e.g. a header comment, PCH inclusion, etc.

Reply:
I don't get this well, sorry! can you help me understand?

Reply:
Just remove the new `.cpp` file and put the (little) code there is into the header.